static/js/components/PrivacyForm.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+
+import ProfileProgressControls from './ProfileProgressControls';
+import ProfileFormFields from '../util/ProfileFormFields';
+import {
+  combineValidators,
+  personalValidation,
+  educationValidation,
+  educationUiValidation,
+  employmentValidation,
+  employmentUiValidation,
+  privacyValidation,
+} from '../util/validation';
+
+class PrivacyForm extends ProfileFormFields {
+  static propTypes = {
+    profile:        React.PropTypes.object,
+    saveProfile:    React.PropTypes.func,
+    updateProfile:  React.PropTypes.func,
+    ui:             React.PropTypes.object
+  };
+
+  render() {
+    const { saveProfile, profile, isSettingPage, ui, updateProfile } = this.props;
+    let controlls = (
+      <ProfileProgressControls
+        prevUrl="/profile/professional"
+        nextUrl="/dashboard"
+        isLastTab={true}
+        saveProfile={saveProfile}
+        profile={profile}
+        ui={ui}
+        validator={
+          combineValidators(
+            personalValidation,
+            educationValidation,
+            educationUiValidation,
+            employmentValidation,
+            employmentUiValidation,
+            privacyValidation
+         )
+        }
+      />
+    );
+    if (isSettingPage) {
+      controlls = (
+        <ProfileProgressControls
+          nextUrl="/dashboard"
+          isLastTab={true}
+          saveProfile={saveProfile}
+          profile={profile}
+          ui={ui}
+          validator={
+            combineValidators(
+              personalValidation,
+              educationValidation,
+              educationUiValidation,
+              employmentValidation,
+              employmentUiValidation,
+              privacyValidation
+            )
+          }
```

Comment:
it might be a little more manageable to deal with the `<ProfileProgressControls>` if you pulled the things we need to be different out into little helper methods, which you could then just call inline in the props we're passing to the `<ProfileProgressControls>` component.
```js
const { isSettingsPage } = this.props;
let prevUrl = () => (
  isSettingsPage ? undefined : "/profile/professional"
);

let validator = () => (